src/processing/2_Verden/Verden.md

```diff
@@ -0,0 +1,492 @@
+---
+title: Verden
```

Comment:
Eg tenkte det gikk greit at de hadde samme navn ettersom de kom til å havne i forskjellige lister, den eine under intro og den andre under nybegynner. Ser i scratch at det er delt inn i del 1, del 2 og del 3, men det er for oppgaver på samme nivå.  

Kan godt skifte, men er det da "Verden - Del 2" som er det vi skal gå for?